### DISCUSSION TITLE
Mover Stepper para o topo da página

### DISCUSSION THREAD

**Discussion by Smmartinez** [po-ui/po-angular#1695] [Tire suas dúvidas] [answered]
Fala galera tudo bem?
Estou utilizando o stpper para criar passos para um checklist. Os campos são apresentados e o botão de próximo passo foca no final da página. E quando mudamos para o próximo passo o formulário não retorna para o inicio ficando sempre no ultimo item, fazendo com o que usuário tenha que rolar a tela para o primeiro item.
Existe alguma forma de fazer o stpper sempre ir para o topo da página?

**Comment by anliben** [comment, marked as answer]
Bom Dia @Smmartinez tudo bem?

Obrigado por reportar, irei fechar a outra discussions, pois está duplicada!

Para nos entendermos melhor sua solicitação, consegue montar um exemplo no stackblitz e nos enviar?

Link: https://stackblitz.com/edit/po-ui

Attr.

**Comment by Smmartinez** [comment]
@anliben bom dia!
Segue o exemplo https://po-ui-wnld4y.stackblitz.io

Acredito que o problema seja no MODAL e não no STEPPER.
Se você for até o final da pagina e clicar em próxima etapa o próximo passo não começa do topo do modal.

Obrigado pela ajuda.

**Reply by anliben** [reply]
Boa Tarde!

Poderia me enviar o stackblitz porem no modo de desenvolvedor para eu editar e não no modo view, pois aí não consigo analisar!

Attr.

**Comment by Smmartinez** [comment]
@anliben voce conseguiu testar o exemplo?

**Comment by Smmartinez** [comment]
Bom dia!
Alguém olha as discussões?

**Comment by anliben** [comment]
@Smmartinez, Boa Tarde, tudo bem?

Desculpe pela falta de respostas, tivemos umas sprints bastante focados em entregas do UX, irei dar retorno, amanha mesmo!

Attr.

**Reply by Smmartinez** [reply]
Opa bom dia!
Conseguiu analisar o problema?
